poetry.lock

```diff
@@ -990,7 +990,7 @@ description = "Converting Jupyter Notebooks"
 name = "nbconvert"
 optional = true
 python-versions = ">=3.6"
-version = "6.0.6"
+version = "6.0.7"
```

Comment:
This here is the cause of the break. It seems that going from version `6.0.6` --> `6.0.7` breaks the docs build.

Reply:
Reported the issue at https://github.com/jupyter/nbconvert/issues/1442.

For now will leave nbconvert pinned to 6.0.6